app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Posts show</h1>
+</section>
+    <div class = "Postpage" >
+        <div  class = "page">
+        <p>Post #1 by<%= @user.name %></p>
+         <p class = "postsCounter" >
+            Comments: <%= @posts.comments_counter %>
+          <%= button_to "* Like", user_post_likes_path(:user_id, :post_id), method: :post %>:<%= @posts.likes.count %>
```

Comment:
- [ ] Kindly make sure when you click on the like button, the number of likes increases by one.